src/main/java/net/fabricmc/loom/configuration/ide/RunConfigSettings.java

```diff
@@ -308,6 +308,11 @@ public void client() {
 		startFirstThread();
 		environment("client");
 		defaultMainClass(Constants.Knot.KNOT_CLIENT);
+
+		if (Platform.CURRENT.isRaspberryPi()) {
+			getProject().getLogger().lifecycle("Raspberry Pi detected, setting MESA_GL_VERSION_OVERRIDE=4.3");
```

Comment:
Nit: Maybe `info` is enough? I'm not sure this needs to be printed out every configuration cycle.

Reply:
Done 👍